### PR TITLE
Module open api end points display

### PR DESCRIPTION
**Pull request by lecunm**

### CODE REVIEW

File: app/controllers/OpenApiController.scala [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import javax.inject._
+import play.api.libs.json.Json
+import play.api.libs.ws.WSClient
+import play.api.mvc._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class OpenApiController @Inject()(ws: WSClient)(implicit ec: ExecutionContext) extends InjectedController {
+
+  @ApiOperation(value = "Get pages from path", response = classOf[Any])
```
Comment (gaeljw) [inline app/controllers/OpenApiController.scala:13]: Swagger documentation looks like a copy paste ;)

File: frontend/src/app/output/page-content/open-api-end-points/open-api-end-points.component.ts [hunk 2]
```diff
@@ -13,9 +14,14 @@ export class OpenApiEndPointsComponent implements OnInit,AfterViewInit {
   @Input() openApiPathJson: OpenApiPath;
   @Input() position: number;
 
-   id:string;
+  id:string;
+  proxyUrl:string;
 
-  constructor() { }
+  constructor(private applicationService: ConfigService) {
+    this.applicationService.getConfigs().subscribe(result => {
```
Comment (gaeljw) [inline frontend/src/app/output/page-content/open-api-end-points/open-api-end-points.component.ts:21]: I would do it on the `ngOnInit`. Waiting @loriepisicchio thought on this but I believe constructor is not the place to call other services.

File: test/services/clients/OpenApiClientTest.scala [hunk 3]
```diff
@@ -36,7 +36,7 @@ class OpenApiClientTest extends WordSpec with MustMatchers with MockitoSugar wit
   val pageJoinProject = PageJoinProject(page, directory, branch, project)
 
   val expectedErrorModel = OpenApiModel("", Option(Seq()), Seq(), Seq(), Seq(" "))
-  val expectedErrorPath = OpenApiPath(Json.parse("{}"), Seq(""))
+  val expectedErrorPath = OpenApiPath(Json.parse("{}"), "",Seq(""))
```
Comment (notflorian) [inline test/services/clients/OpenApiClientTest.scala:39]: Format issue
```suggestion
  val expectedErrorPath = OpenApiPath(Json.parse("{}"), "", Seq(""))
```

File: frontend/src/app/output/page-content/open-api-model/open-api-model.component.ts [hunk 4]
```diff
@@ -20,7 +20,9 @@ export class OpenApiModelComponent implements OnInit {
   }
 
   isRequired(element: OpenApiModelRow) {
-    return this.openApiModule.required.includes(element.title);
+
```
Comment (loriepisicchio) [inline frontend/src/app/output/page-content/open-api-model/open-api-model.component.ts:23]: :space_invader:

**Pull request merged**